ui-core/src/main/java/com/adyen/checkout/ui/core/internal/util/CustomTabsLauncher.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2024 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 6/5/2024.
+ */
+
+package com.adyen.checkout.ui.core.internal.util
+
+import android.content.ActivityNotFoundException
+import android.content.Context
+import android.content.res.TypedArray
+import android.net.Uri
+import androidx.annotation.RestrictTo
+import androidx.annotation.StyleableRes
+import androidx.browser.customtabs.CustomTabColorSchemeParams
+import androidx.browser.customtabs.CustomTabsIntent
+import com.adyen.checkout.ui.core.R
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+object CustomTabsLauncher {
+    fun launchCustomTab(context: Context, uri: Uri): Boolean {
+        @Suppress("SwallowedException")
+        return try {
+            CustomTabsIntent.Builder()
+                .setShowTitle(true)
+                .setDefaultColorSchemeParams(getDefaultColorSchemeParams(context))
+                .build()
+                .launchUrl(context, uri)
+            true
+        } catch (e: ActivityNotFoundException) {
```

Comment:
Should we log this exception?

Reply:
Right now its callers are logging it based on their own behavior, some of them fall back to another solution (so log level is debug) while other fail without it (log level error). Do you think we should add a verbose log here as well?

Reply:
I was thinking that the exception maybe has some useful information, but on second thought probably not. When this exception is thrown it basically can only mean there is no compatible browser. So I think it's fine like this